formats/simplejsonapi.go

```diff
@@ -96,6 +96,7 @@ type CveRow struct {
 type Applicability struct {
 	Status             string     `json:"status"`
 	ScannerDescription string     `json:"scannerDescription,omitempty"`
+	UndeterminedReason string     `json:"undeterminedReason,omitempty"`
```

Comment:
where do we use it? `UndeterminedReason` is being inserted into - but not called...

Reply:
It is only in simple-json, and it is under CveRow (CveRow has Applicability struct as one of its components, it gets printed automatically)